cnames_active.js

```diff
@@ -3381,6 +3381,7 @@ var cnames_active = {
   "vlite": "vlitejs.github.io",
   "vlr": "vlrjs.github.io/website",
   "vncz": "xvincentx.github.io/vncz",
+  "vnpay": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "vnpay": "cname.vercel-dns.com", // noCF
```
As the CI error notes, you'll want the Cloudflare proxy disabled when using Vercel

Reply:
I added it